lib/types/json.js

```diff
@@ -77,10 +77,13 @@ function json(options) {
 
     if (strict) {
       var first = firstchar(body)
-
+      var message = 'Unexpected token ' + first
       if (first !== '{' && first !== '[') {
+        if (process.version[1] >= 6) {
+          message += ' in JSON at position 1'
```

Comment:
I was under the impression this throw was only for the first character based on the boolean logic just above